app/pods/publications/publication/documents/template.hbs

```diff
@@ -461,10 +461,11 @@
 {{!-- TODO replace this with Au variant --}}
 {{#if this.isVerifyingDelete}}
   <WebComponents::VlModalVerify
-    @title={{t "document-delete"}}
-    @message={{t "delete-document-message"}}
-    @cancel={{action this.cancelDeleteExistingPiece}}
-    @verify={{perform this.verifyDeleteExistingPiece}} />
+          @title={{t "document-delete" }}
```

Comment:
Gewoon een stijlvraag ... Waarom doen we dit de laatste tijd steeds? We werken toch overal met 2-space indent?

Reply:
In de `hbs` files lijken de editors wat moeilijk te doen met indentaties is mij al opgevallen dat de `template-linter` ook niet alles m.b.t. octane even hard afchecked.. misschien loop de dependency wel nog achter.. iets wat we moeten checken.